### PR TITLE
WAZO-1375 csv backend: log errors when an absent column is configured

### PR DESCRIPTION
**Pull request by pc-m**
If a column is configured in searched_columns, unique_column,
first_match_columns and that column does not exist dird will ignore it during a
lookup and log an error message.

### CODE REVIEW

**Comment by pc-m** [comment]
recheck

File: wazo_dird/plugins/source_result.py [hunk 1]
```diff
@@ -64,9 +64,10 @@ def get_unique(self):
         try:
             return str(self.fields[self._unique_column])
         except KeyError:
-            logger.exception(
-                '%s is not properly configured, the unique column is not part of the result',
+            logger.info(
```
Comment (sduthil) [inline wazo_dird/plugins/source_result.py]: Why `logger.info`? How about `logger.error`?

**Comment by pc-m** [comment]
@sduthil are there anything missing in this PR to include in the current sprint?

**Comment by sduthil** [comment]
> @sduthil are there anything missing in this PR to include in the current sprint?

Nothing missing, I just wanted some time to play with it, but could not find the time.

**Pull request merged**